ui/src/main/java/com/xmartlabs/bigbang/ui/BaseAppCompatActivity.java

```diff
@@ -39,13 +36,4 @@ protected void onCreate(Bundle savedInstanceState) {
   public void removeFragment(@NonNull Fragment fragment) {
     getSupportFragmentManager().beginTransaction().remove(fragment).commit();
   }
-
```

Comment:
I think you shouln't remove it. You can add another in the fragment, but you shouldn't remove it

Reply:
You're right, I will add it again